packages/sdk/schema/src/common/message.ts

```diff
@@ -122,6 +123,42 @@ export namespace ContentBlock {
   }).pipe(Schema.mutable);
   export interface ToolResult extends Schema.Schema.Type<typeof ToolResult> {}
 
+  /**
+   * GPT Summary
+   */
+  export const Summary = Schema.TaggedStruct('summary', {
+    mimeType: Schema.optional(Schema.String),
+    message: Schema.String,
+    usage: Schema.optional(
```

Comment:
I'd rather include the usage under eath message with `role=assistant`

Reply:
to avoid having a new message or to have greater preceision (which you have to reduce)? i don't see the disadvantage in having a message for really important stuff like this.